tululbot/commands.py

```diff
@@ -65,6 +65,20 @@ def hotline(message):
         return bot.forward_message(message.chat.id, message.chat.id, HOTLINE_MESSAGE_ID)
 
 
+@bot.message_handler(is_reply_to_bot='Siapa yang ultah?')
+@bot.message_handler(commands=['hbd'])
+def hbd(message):
+    app.logger.debug('Detected as hbd command')
+    term = _extract_birthday_boy_or_girl_name(message)
+    if not term:
+        return bot.reply_to(message, 'Siapa yang ultah?',
+                            force_reply=True)
+    else:
+        greetings = "hoi " + term \
+            + " met ultah ya moga sehat dan sukses selalu :tada: :confetti_ball:"
+        return bot.reply_to(message, greetings)
```

Comment:
- [ ] Instead of replying to the message, can we have the bot to simply send the message to the chat (group)?